src/eventloop.jl

```diff
@@ -18,7 +18,15 @@ function eventloop(socket)
                 end
             finally
                 @async begin
-                    sleep(idle_delay[])
+                    if check_interact_loaded() && idle_delay[] < interact_min_delay
```

Comment:
So this was intended to be a deprecation sort of warning, where for the time being, I won't break Interact.jl, and this PR can get processed (as I haven't even looked at Interact's source to patch it.)

I promise someone can remove it a version or two down the line.